app/renderer/js/spellchecker.js

```diff
@@ -26,13 +26,13 @@ class SetupSpellChecker {
 		}
 	}
 
-	enableContextMenu() {
+	enableContextMenu(serverLanguage) {
 		if (this.SpellCheckHandler) {
 			this.SpellCheckHandler.attachToInput();
-
-			const userLanguage = ConfigUtil.getConfigItem('spellcheckerLanguage');
-
-			this.SpellCheckHandler.switchLanguage(userLanguage);
+			this.SpellCheckHandler.switchLanguage(serverLanguage);
+			this.SpellCheckHandler.currentSpellcheckerChanged.subscribe(() => {
```

Comment:
@kanishk98  Can you help me understand what this does? I need to know this for BrowserView. Getting problems due to this.